src/blocks/time.rs

```diff
@@ -38,17 +38,29 @@ pub struct Config {
     format: FormatConfig,
     #[default(1.into())]
     interval: Seconds,
-    timezone: Option<Tz>,
+    timezones: Option<Vec<Tz>>,
```

Comment:
This is a breaking change but it doesn't need to be. See #1452 for very similar example. TL;DR: You can create an enum `Timezone` which is either `Tz` or `Vec<Tz>`. `#[serde(untagged)]` will handle the deserialization.